Source/Blazorise.Tailwind/Check.razor

```diff
@@ -2,6 +2,9 @@
 @inherits Blazorise.Check<TValue>
 <Control Role="ControlRole.Check" Inline="@Inline">
     <input @ref="@ElementRef" id="@ElementId" type="checkbox" class="@ClassNames" style="@StyleNames" disabled="@Disabled" readonly="@ReadOnly" checked="@Checked" tabindex="@TabIndex" @onchange="@OnChangeHandler" @onkeydown="@OnKeyDownHandler" @onkeypress="@OnKeyPressHandler" @onkeyup="@OnKeyUpHandler" @onblur="@OnBlurHandler" @onfocus="@OnFocusHandler" @onfocusin="@OnFocusInHandler" @onfocusout="@OnFocusOutHandler" @attributes="@Attributes" />
-    <Label Type="LabelType.Check" For="@ElementId" Style="@Style" Cursor="@Cursor">@ChildContent</Label>
+    @if ( ChildContent is not null )
+    {
+        <Label Type="LabelType.Check" For="@ElementId" Style="@Style" Cursor="@Cursor">@ChildContent</Label>
+    }
```

Comment:
It seems that it is OK from what I can see. We might do the same for other providers and see how that will work. It can be usable on DataGrid for multi select where we also don't need the label.